### ISSUE TITLE
[BUG REPORT] - videos not playing on "Screen 2"  for 2-monitor setup on Win11

### ISSUE THREAD

**Issue by nazzy121** [SegoCode/AutoWall#31]
### Description

I believe this bug is specific to Windows 11 because both v1.11 and v1.12 Autowall can't run videos on 2 monitors, videos only play on Monitor 1,  never on Monitor 2.    However, v1.11 can successfully run videos on 2 monitors in Win 10.  

### Steps to reproduce

1. Be using Win 11 OS. 
2. Click "yes" when asked to run in Multi-Screen Mode
3. Select "Apply to Screen 1",  choose video.  Video plays on Screen 1. 
4. Select "Apply to Screen 2".  choose video.  Video plays on Screen 1.  

Basically, the video never plays on Screen 2. 

### Expected behavior

Expect video to play on Screen 2 when "Apply to Screen 2" is selected. 

### Actual behavior

Videos only play on "Screen 1". 

### Windows version

Windows 11

### Confirmation

- [X] I performed a [search of the issue tracker](https://github.com/segocode/AutoWall/issues) to avoid opening a duplicate issue
- [X] I understand that not filling out this template correctly may lead to the issue being closed

**Comment by SegoCode** [comment]
I see, thanks for reporting the issue, next update I want to focus on fixing all the current multiscreen problems, added to the todo list.

**Comment by nazzy121** [comment]
> I see, thanks for reporting the issue, next update I want to focus on fixing all the current multiscreen problems, added to the todo list.

Sounds good, and THANK YOU for removing the "no file was selected" pop-up message that was in previous version when you closed the file explorer without actually selecting a file.

**Comment by SegoCode** [comment]
> Sounds good, and THANK YOU for removing the "no file was selected" pop-up message that was in previous version when you closed the file explorer without actually selecting a file.

Also you can now turn off the multiscreen popup https://github.com/SegoCode/AutoWall#Config-autowall if you need it 😄

**Comment by SegoCode** [comment]
Nope 😅
